### PR TITLE
release(v0.2.0): prepare release

### PR DESCRIPTION
**Pull request by Unix4ever**
This is the official v0.2.0 release.

Signed-off-by: Artem Chernyshev <artem.chernyshev@talos-systems.com>

### CODE REVIEW

**Comment by smira** [comment]
@Unix4ever not `master`, you should get this to `release-0.2` branch

**Comment by Unix4ever** [comment]
> @Unix4ever not master, you should get this to release-0.2 branch

Always forget to pick the right base in these cases. Now it should be ok.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,3 +1,141 @@
+## [CAPI Control Plane Provider Talos 0.2.0](https://github.com/talos-systems/cluster-api-control-plane-provider-talos/releases/tag/v0.2.0) (2021-09-24)
+
+Welcome to the v0.2.0 release of CAPI Control Plane Provider Talos!  
+*This is a pre-release of CAPI Control Plane Provider Talos*
```
Comment (smira) [inline CHANGELOG.md:4]: please change `pre-release` in hack/release.toml

**Review by smira — APPROVED** [review]

**Pull request merged**